tasks/install-alert_manager.yml

```diff
@@ -1,5 +1,18 @@
 - include_vars: 'alertmanager.yml'
 
+- name: "Creating directories"
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: "{{ prometheus_owner }}"
+    group: "{{ prometheus_group }}"
+    recurse: yes
+    mode: 0750
+  with_items:
+    - "{{ prometheus_alert_manager_data_dir }}"
```

Comment:
prometheus_alert_manager_data_dir: "{{ prometheus_lib_dir }}/alertmanager" requires `prometheus_lib_dir` please add it as well